src/staking/manager.ts

```diff
@@ -1123,3 +1481,53 @@ export const getUnbondingTxStakerSignature = (
     );
   }
 };
+
+/**
+ * Validates the staking expansion input
```

Comment:
Do we aim to validate everything here? For example, we are not checking, whether finality providers are from different BSNs but not sure if that's the place.